src/main/scala/tech/cryptonomic/conseil/routes/Metadata.scala

```diff
@@ -0,0 +1,66 @@
+package tech.cryptonomic.conseil.routes
+
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import com.typesafe.config.Config
+import com.typesafe.scalalogging.LazyLogging
+import tech.cryptonomic.conseil.tezos.TezosMetadataOperations
+import tech.cryptonomic.conseil.util.JsonUtil._
+import tech.cryptonomic.conseil.util.{ConfigUtil, RouteHandling}
+
+import scala.concurrent.ExecutionContext
+
+/** Companion object providing apply implementation */
+object Metadata {
+  def apply(config: Config)(implicit apiExecutionContext: ExecutionContext): Metadata =
+    new Metadata(config)
+}
+
+/**
+  * Platform discovery routes.
+  *
+  * @param config              configuration object
+  * @param apiExecutionContext is used to call the async operations exposed by the api service
+  */
+class Metadata(config: Config)
```

Comment:
As previously discussed, it might be better to rename this route as `PlatformDiscovery` and combine it with any eventual new metadata endpoints using akka-http routing api.
Currently this "route module" is neatly focused on platforms, so keeping separate concerns in specific modules seems a good idea.
Any thoughts or counter-arguments?

Reply:
renamed